### DISCUSSION TITLE
How install ESRGAN?

### DISCUSSION THREAD

**Discussion by CruvinelJP** [community/community#137370] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Excuse me for asking, but I'm very new to programming and I find github to be absurdly unintuitive. Despite that, I love photography and the other day I was browsing my social media and an ad from Qualcomm appeared about their AI, more specifically an ERSGAN service to improve image quality and remove noise. Here's the link to see: https://aihub.qualcomm.com/models/esrgan. 
My S24 Ultra device is compatible because it has a Snapdragon 8 gen 3, but it's not just a matter of downloading, because when the download is complete I don't have an app that can open/extract the file (it certainly won't be my graphing calculator or health app that can do that...). I tried searching the web for what to do but I didn't find much information. Everything was very Windows-oriented and in a language that was a bit confusing to me. What does Python have to do with all this??? Well, when I looked for more information about this Qualcomm AI HUB, it directed me to GitHub, so I opened this discussion in the hope that someone knows how to deal with this, because no one on the Qualcomm subreddit could answer me. 
Now tell me how to make an announcement for ordinary people, since soon I'll have to go to the Qualcomm lab to get an idea of ​​what to do.

**Comment by xsxo** [comment]
i think you will find the answer on reddit, if you can use it write your question there

**Reply by CruvinelJP** [reply]
I've tried there but no one knew how to answer me how to download this apk
